recipes/config.rb

```diff
@@ -23,17 +23,17 @@
 # See libraries/helpers for the PushJobsHelper module.
 directory PushJobsHelper.config_dir do
   unless platform_family?('windows')
-    owner 'root'
```

Comment:
We should just remove the owner/group since root / root is the default and we'll pick the right one based on platform

Reply:
Does this default to "0" on AIX and not "root"? I no longer have an AIX test platform to confirm that.

Reply:
It should be doing the right thing on AIX just like we do the right thing on other *nix releases.